test_fms/axis_utils/test_axis_utils2.sh

```diff
@@ -8,9 +8,14 @@
 # Set common test settings.
 . ../test_common.sh
 
+# If there is a mpi launcher set the number of processors to 2, otherwise just ./
+if test "$mpi_launcher" != "" ; then 
+    npes="-n 2"
```

Comment:
Since libFMS _can_ be compiled w/o MPI, we need to determine which tests can be run in that case.  Please open an issue to remind us of this.